### PR TITLE
Add recipe for watchdog-gevent

### PR DESCRIPTION
**Pull request by lincrosenbach**

### CODE REVIEW

**Comment by lincrosenbach** [comment]
Open an issue for the missing LICENSE file on Pypi. Here is the link:
https://github.com/Bogdanp/watchdog_gevent/issues/1

File: recipes/watchdog-gevent/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  license: Apache Version 2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: A gevent-based observer for watchdog.
+  dev_url: https://github.com/Bogdanp/watchdog_gevent
+
+extra:
+  recipe-maintainers:
+    - lincrosenbach
+    - nicoddemus
```
Comment (nicoddemus) [inline recipes/watchdog-gevent/meta.yaml]: 👍

File: recipes/watchdog-gevent/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - gevent >=1.1
+    - pip
```
Comment (nicoddemus) [inline recipes/watchdog-gevent/meta.yaml]: You probably only need `pip` and `python` in this section.

**Comment by tadeu** [comment]
There is a warning when importing it in the TravisCI build:

```
import: 'watchdog_gevent'
/Users/travis/miniconda/conda-bld/watchdog-gevent_1538497986009/_test_env_placehold(...)/lib/python3.6/site-packages/watchdog/observers/__init__.py:74: UserWarning: Failed to import fsevents. Fall back to kqueue
```

**Comment by nicoddemus** [comment]
> There is a warning when importing it in the TravisCI build (OSX):

I don't think it is a problem, it is an optional dependency on Mac.

File: recipes/watchdog-gevent/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    - gevent >=1.1
+    - python
+    - watchdog >=0.8
+
+test:
+  imports:
+    - watchdog_gevent
+
+about:
+  home: The package home page
```
Comment (synapticarbors) [inline recipes/watchdog-gevent/meta.yaml]: Please list and actual url for `home`

File: recipes/watchdog-gevent/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  license: Apache Version 2.0
```
Comment (synapticarbors) [inline recipes/watchdog-gevent/meta.yaml]: `license: APACHE-2.0`

File: recipes/watchdog-gevent/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (synapticarbors) [inline recipes/watchdog-gevent/meta.yaml]: Can you add a comment with the link to your PR in the meta.yaml` here, to add the `license_file` to the sdist just as a reminder for the future once that PR is potential merged

File: recipes/watchdog-gevent/meta.yaml (hunk 3 above)
Comment (lincrosenbach) [reply recipes/watchdog-gevent/meta.yaml]: Filled with the github page.

File: recipes/watchdog-gevent/meta.yaml (hunk 4 above)
Comment (lincrosenbach) [reply recipes/watchdog-gevent/meta.yaml]: Renamed.

File: recipes/watchdog-gevent/meta.yaml (hunk 5 above)
Comment (lincrosenbach) [reply recipes/watchdog-gevent/meta.yaml]: Added comment with link to issue.

**Comment by synapticarbors** [comment]
I'm happy to merge this once the CI finish up. Please feel free to ping me if it's completed and I haven't noticed yet.

File: recipes/watchdog-gevent/meta.yaml (hunk 2 above)
Comment (lincrosenbach) [reply recipes/watchdog-gevent/meta.yaml]: Thanks.

**Pull request merged**